politeiawww/user/mysql/mysql.go

```diff
@@ -80,8 +81,15 @@ const tableSessions = `
   sBlob BLOB NOT NULL
 `
 
+// tableEmailHistories defines the email_histories table.
+const tableEmailHistories = `
```

Comment:
See the naming convention explination from this issue.
https://github.com/decred/politeia/issues/1457